### DISCUSSION TITLE
Can `Produce` support the App Store Connect API?

### DISCUSSION THREAD

**Discussion by cdoky** [fastlane/fastlane#19065] [General]
Can `Produce` support the App Store Connect API?

**Comment by crazymanish** [comment]
It supports **Partial** https://docs.fastlane.tools/app-store-connect-api/
.
<img width="624" alt="Screenshot 2021-07-10 at 21 22 15" src="https://user-images.githubusercontent.com/5364500/125174328-f2e70c80-e1c4-11eb-8a7f-b42fc9ac1f11.png">

**Reply by cdoky** [reply]
Thank you!
I mean, now Produce does not support APIKEY. Is APIKEY itself does not support the function of Produce or is it because FastLane has not integrated APIKEY into the Produce module for the time being?

**Reply by crazymanish** [reply]
> Is APIKEY itself does not support the function of Produce

This 👆, **Apple does not support all the features yet using API key**, that's why Fastlane can't support all features using API key
